flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/analyze/NonDeterministicUpdateAnalyzerTest.java

```diff
@@ -356,6 +357,92 @@ void testMultiSinkOnJoinedView() {
                 false);
     }
 
+    @Test
+    void testMultiSinkOnMultiJoinedView() {
+        util.getTableEnv().getConfig().set(TABLE_OPTIMIZER_MULTI_JOIN_ENABLED, true);
```

Comment:
nit: instead of calling `util.getTableEnv()` many times. introduce a local variable `env`